crates/turborepo-lib/src/cli.rs

```diff
@@ -264,6 +264,14 @@ pub enum Command {
         #[clap(long, value_enum, default_value_t = LinkTarget::RemoteCache)]
         target: LinkTarget,
     },
+    /// Generate a new app / package
```

Comment:
Why G instead of Generate?

Reply:
I'm fine with generate, G was the original name suggested by @nathanhammond so was rolling with it.

Reply:
I figured it was a shout out to @rauchg haha

Reply:
Well now we definitely need that 😂

Reply:
@NicholasLYang updated

```
local-turbo generate --help
Generate a new app / package

Usage: turbo generate [OPTIONS] <COMMAND>

Commands:
  add  Add a new package or app to your project
  run  Run custom generators
```